frontend/pulse3d/src/components/layouts/DashboardLayout.js

```diff
@@ -67,18 +67,18 @@ export default function DashboardLayout({ children }) {
       setMetaPulse3dVersions(versions); // keep track of states
 
       const testingVersions = versions.filter(({ state }) => state === "testing");
-      const externalVersions = versions.filter(({ state }) => state === "external");
-
+      const externalVersions = versions.filter(({ state }) => state === "external" || state === "deprecated");
```

Comment:
We should change the name of this to something else now that it is not only the external versions, maybe something like `supportedVersions`